deploy/helm/magda-core/requirements.yaml

```diff
@@ -39,10 +39,10 @@ dependencies:
     tags:
       - all
       - indexer
-  - name: preview-map
-    tags:
-      - all
-      - preview-map
+  # - name: preview-map
```

Comment:
We still need map preview -- just need to reference it from http://charts.magda.io

Reply:
Oops. Forgot to do that :+1: